Makefile

```diff
@@ -212,6 +242,22 @@ $(FOGBENCH_PYTHON_INSTALL_DIR) :
 $(EXTRAS_INSTALL_DIR) :
 	$(MKDIR_PATH) $@
 
+###############################################################################
+####################### DATA INSTALL TARGETS ################################
+###############################################################################
+# install data
+data_install : $(DATA_INSTALL_DIR) install_data
+
+install_data : $(DATA_INSTALL_DIR) $(DATA_SRC_DIR)
+	$(CP_DIR) $(DATA_SRC_DIR) $(INSTALL_DIR)
+ifdef SUDO_USER
+	chown -R ${SUDO_USER}:${SUDO_USER} $(INSTALL_DIR)/$(DATA_SRC_DIR)
```

Comment:
chown is causing a conflict on ubuntu core snap creation.

Reply:
Ordinarily the chown would be done as part of an install. I can see why you want to do it here, but maybe make it conditional or ignore error returns for the snap creation case